lms/djangoapps/teams/tests/test_views.py

```diff
@@ -791,6 +801,60 @@ def test_member_trying_to_get_private_topic(self):
         self.assertEqual('private_topic_1_id', result['results'][0]['topic_id'])
         self.assertNotEqual([], result['results'])
 
+    @ddt.unpack
+    @ddt.data(
+        ('student_masters_not_on_team', True),
+        ('student_masters', True),
+        ('student_enrolled', False),
+        ('staff', True),
+    )
+    def test_text_search_organization_protected(self, user, can_see_masters_team):
+        """
+        When doing a text search as different users, will the masters_only team show up?
+        Only staff, or people who are within the organization_protected bubble should be
+        able to see the masters team
+        """
+        self.reset_search_index()
+        result = self.get_teams_list(
+            data={'text_search': 'master'},
+            user=user,
+        )
+        expected_teams = 1 if can_see_masters_team else 0
```

Comment:
For can_see_masters_team argument: Call it expected_team_count and pass in 1 or 0?